network.go

```diff
@@ -62,6 +62,7 @@ type NetworkInfo interface {
 	IpamInfo() ([]*IpamInfo, []*IpamInfo)
 	DriverOptions() map[string]string
 	Scope() string
+	EnableIPv6() bool
```

Comment:
I was on the fence - I wanted to strive for consistency (e.g. Internal returns internal and it's called EnableIPv6 throughout the change in docker/docker), but it does sound like a command name. I've made the change as requested.